languages/java/config.json

```
@@ -57,6 +57,12 @@
         "uuid": "2165c331-6b0c-49b8-9555-bad577ec4fc6",
         "concepts": ["strings"],
         "prerequisites": ["basics"]
+      },
+      {
+        "slug": "conditionals-ternary",
+        "uuid": "2165c331-6b0c-49b8-9555-bad577ec4fc6",
+        "concepts": ["strings"],
```

This key should be the concepts this exercise unlocks:

```suggestion
        "concepts": ["ternary-operators"],
```